tests/phpunit/unit/Extension/TwigTraitTest.php

```diff
@@ -11,7 +13,44 @@
  */
 class TwigTraitTest extends BoltUnitTest
 {
-    public function test()
+    public static function setUpBeforeClass()
     {
+        parent::setUpBeforeClass();
+        $fs = new Filesystem();
+        $fs->copy(PHPUNIT_ROOT . '/resources/test.twig', PHPUNIT_WEBROOT . '/theme/base-2016/test.twig', true);
+    }
```

Comment:
Can you just add an `Array_Loader` to the `Twig_Loader_Chain` for that render test?
The less files we have to copy around the better.